### ISSUE TITLE
[Feature request] Support for Xbox through new Mesa UWP port

### ISSUE THREAD

**Issue by tleerai** [RPCS3/rpcs3#15787]
### Quick summary

Mesa is being ported to UWP, would like to see RPCS3 support it. 

https://github.com/aerisarn/mesa-uwp

### Details

Since there is now a working Mesa port to UWP I think that removes one of the main hurdles to getting RPCS3 on the Xbox.

Check it out here:

**Comment by FlexBy420** [comment]
It doesn't change the fact that theres not enough ram in dev mode lol

**Comment by AniLeo** [comment]
Duplicate of #9315, not of interest and still blocked by other dev mode restrictions

**Comment by edvinkuric** [comment]
Hello, could you point me to any resources or provide guidance on what is necessary to make RPCS3 run as an UWP app for the XBox Series X?

Does Qt need to be removed as a whole?
Does MESA need to be someehow activated specifically for the xbox port?
What else would be necessary.

I got a xbox series x and would love to somehow test this, if it isn't too troublesome.
I also got some programming experience with Windows Applications

Any help would be appreciated :-)
BR

**Comment by skast96** [comment]
@edvinkuric did you manage to run it on your xbox. I want to try it myself as well.

**Comment by wicked-313** [comment]
The Xbox can run PS3 emulation it has the ram for it... The Xbox series x has 16GB of ram the whole PS3 system only has about 600mb of ram roughly 300mb of ram and 300mb of vram... And rpcs3 recommended 16GB of ram and when you remember that consoles like the Xbox have external ssds it's not out of the realm of possibility and when you realize that chonky station 3 is already on Xbox series x in alpha state already it only proves it can be done. @FlexBy420

**Comment by Gael2433** [comment]
> The Xbox can run PS3 emulation it has the ram for it... The Xbox series x has 16GB of ram the whole PS3 system only has about 600mb of ram roughly 300mb of ram and 300mb of vram... And rpcs3 recommended 16GB of ram and when you remember that consoles like the Xbox have external ssds it's not out of the realm of possibility and when you realize that chonky station 3 is already on Xbox series x in alpha state already it only proves it can be done. [@FlexBy420](https://github.com/FlexBy420)

Dev mode restricts the amount of memory a program can use to about 4Gb which is not enough(nor is the AMD APU in the system powerful enough to run rpcs3).
